tests_v2/unit/app/routes/geostore/test_geostore.py

```diff
@@ -178,7 +184,95 @@
 
 
 @pytest.mark.asyncio
-async def test_get_admin_geostore(async_client: AsyncClient, monkeypatch: MonkeyPatch):
+async def test_get_admin_country_geostore_gfw_branch(
+    async_client: AsyncClient, monkeypatch: MonkeyPatch
+):
+    # CursorResult = MagicMock(cursor)
+    # mock_session.execute.return_value = [
+    #     CursorResult(tup1),
+    #     CursorResult(tup2)
+    # ]
+    # CursorResult.return_value._asdict.side_effect = [tup1, tup2]
+
+    mock_get_first_row: AsyncMock = AsyncMock(return_value=None)
+
+    monkeypatch.setattr(crud_geostore, "get_first_row", mock_get_first_row)
```

Comment:
Why are you mixing monkeypatch and mock library instead of just using the mock patch method?

Reply:
A good question. Will switch to 100% mock.